backend/src/Squidex/Areas/Api/Controllers/Schemas/Models/SchemaDto.cs

```diff
@@ -98,7 +98,7 @@ protected virtual void CreateLinks(Resources resources)
 
             AddSelfLink(resources.Url<SchemasController>(x => nameof(x.GetSchema), values));
 
-            if (resources.CanReadContent(Name))
+            if (resources.CanReadOwnContent(Name))
```

Comment:
This is not necessary, we only have to update the permissions in the resources file and keep everything as it is. It is the responsibility of the Resources class to abstract this logic away.